README.md

```diff
@@ -31,3 +31,25 @@ This is an HTTP server which responds with `Hello World` to every request.
     $socket->listen(1337);
     $loop->run();
 ```
+
+## FormParserFactory and DeferredStream Usage
+
+The `FormParserFactory` parses a quest and determines which body parser to use (multipart, formurlencoded, raw body, or no body). Those body parsers emit events on `post` fields, `file` on files, and raw body emits `body` when it received the whole body. `DeferredStream` listens for those events and returns them through a promise when done.
```

Comment:
Look at it from this angle, I'm on a quest adding streaming request bodies to `react/http` :stuck_out_tongue_closed_eyes: 

Thanks, will fix it :+1: